### PR TITLE
add willit.build blog post

### PR DESCRIPTION
**Pull request by mgienow**

### CODE REVIEW

File: docs/blog/2020-05-26-introducing-will-it-build/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+For those curious as to how it all works, we have open sourced [the example benchmark sites](https://github.com/gatsbyjs/gatsby/tree/master/benchmarks) as well as [the benchmark plugin](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-benchmark-reporting) that reports back the results. Additionally, all of our data can be queried right on the site using our exposed GraphQL API at [willit.build/api-playground](https://willit.build/api-playground).
+
+One of the coolest things about WIllit.build is how clearly it shows the drastic reduction in build times that our new [Incremental Builds](https://www.gatsbyjs.org/blog/2020-04-22-announcing-incremental-builds/) feature makes possible on Gatsby Cloud:
+
+![](https://lh3.googleusercontent.com/9bzilTs1JiTr_VGOuDktdNZ4G-GxBKxot1nX4fdXngUo-CDVrQMSmdaUMnJM7w_Loj6-ncxuv71ebaC5h0_g56tSL6tP6P8P2OxWyA_SbczAFMnc0LLgQkdw35ffuEFvfYUAfk3p)
```
Comment (LekoArts) [inline docs/blog/2020-05-26-introducing-will-it-build/index.md:23]: Alt text needed here :)

Comment (mgienow) [reply docs/blog/2020-05-26-introducing-will-it-build/index.md]: good catch! this is what happens when i start work before coffee

File: docs/blog/2020-05-26-introducing-will-it-build/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+For those curious as to how it all works, we have open sourced [the example benchmark sites](https://github.com/gatsbyjs/gatsby/tree/master/benchmarks) as well as [the benchmark plugin](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-benchmark-reporting) that reports back the results. Additionally, all of our data can be queried right on the site using our exposed GraphQL API at [willit.build/api-playground](https://willit.build/api-playground).
+
+One of the coolest things about WIllit.build is how clearly it shows the drastic reduction in build times that our new [Incremental Builds](https://www.gatsbyjs.org/blog/2020-04-22-announcing-incremental-builds/) feature makes possible on Gatsby Cloud:
```
Comment (jamo) [inline docs/blog/2020-05-26-introducing-will-it-build/index.md]:
```suggestion
One of the coolest things about Willit.build is how clearly it shows the drastic reduction in build times that our new [Incremental Builds](https://www.gatsbyjs.org/blog/2020-04-22-announcing-incremental-builds/) feature makes possible on Gatsby Cloud:
```
or should the first w also be lowercase?

**Comment by muescha** [comment]
where is the post gone? i don't see a new PR...

**Comment by muescha** [comment]
ok - found it here without pr: 
https://github.com/gatsbyjs/gatsby/commits/master/docs/blog/2020-05-26-introducing-will-it-build/index.md

**Pull request closed without merging**